api/src/controllers/profile.ts

```diff
@@ -89,18 +90,21 @@ export const updateProfileContacts = async (
     const currentContacts: Contact[] = await ContactModel.findForProject(Number(profileId));
     const deletedOrNewContact: boolean = true;
     const provisionerContactEdit: boolean[] = [];
+    const removeExistingContact = currentContacts.filter(comparerContact(contacts));
+    const addNewContact = contacts.filter(comparerContact(currentContacts));
+    const AddOrRemoveContact = removeExistingContact.concat(addNewContact);
 
+    if(AddOrRemoveContact.length !== 0){
+      provisionerContactEdit.push(deletedOrNewContact)
+    }
     // 1. Check for provisioner related changes
     contacts.forEach((contact: Contact): void => {
```

Comment:
Nit pick, move the comment to line 96 (include appropriate spacing

Also, would it be worth adding this as an `else` to the if condition above as it would save the additional processing if you know it is already a provisioner edit request.